_blog.yml

```
@@ -5202,4 +5202,15 @@
     - nlp
     - llm
     - community
+    - research
+
```

```suggestion

- local: bamba
  title: "Bamba: Inference-Efficient Hybrid Mamba2 Model"
  author: rganti
  guest: true
  thumbnail: /blog/assets/bamba/bamba_thumbnail.png
  date: Dec 18, 2024
  tags:
    - research
    - nlp
    - community
    - model

```

Attempt to fix conflict.